corehq/apps/hqadmin/tasks.py

```diff
@@ -201,3 +204,39 @@ def track_pg_limits():
                 cursor.execute(f'select last_value from "{sequence}"')
                 current_value = cursor.fetchone()[0]
                 metrics_gauge('commcare.postgres.sequence.current_value', current_value, {'table': table, 'database': db})
+
+
+@periodic_task(queue='background_queue', run_every=crontab(minute="0", hour="4"))
+def reconcile_es_cases():
+    _reconcile_es_data('case', 'commcare.elasticsearch.stale_cases', 'reconcile_es_cases')
+
+
+@periodic_task(queue='background_queue', run_every=crontab(minute="0", hour="4"))
+def reconcile_es_forms():
+    _reconcile_es_data('form', 'commcare.elasticsearch.stale_forms', 'reconcile_es_forms')
+
+
+def _reconcile_es_data(data_type, metric, blob_parent_id):
+    today = date.today()
+    two_days_ago = today - timedelta(days=2)
+    with NamedTemporaryFile('w', delete=False) as output_file:
+        file_name = output_file.name
+        call_command('stale_data_in_es', data_type, start=two_days_ago.isoformat(), stdout=output_file)
+    with open(file_name, 'r') as f:
+        line_count = sum(1 for line in f)
+        # ignore the headers
+        stale_docs = line_count - 1
+        metrics_gauge(metric, stale_docs)
```

Comment:
> One way to do that is to keep the two day window, but also have a two-day window before it that we also sweep that we always expect to be 0 because it consists solely of already-fixed docs

I think this is a great idea. I can add that.